lib/pushmi_pullyu.rb

```diff
@@ -1,12 +1,63 @@
 require 'pushmi_pullyu/logging'
 
 require 'pushmi_pullyu/cli'
-require 'pushmi_pullyu/config'
 require 'pushmi_pullyu/preservation_queue'
 require 'pushmi_pullyu/version'
 require 'active_support'
 require 'active_support/core_ext'
 
 # PushmiPullyu main module
 module PushmiPullyu
+  LOGFILE = 'log/pushmi_pullyu.log'.freeze
+  PIDDIR  = 'tmp/pids'.freeze
+  PROCESS_NAME = 'pushmi_pullyu'.freeze
+
+  DEFAULTS = {
+    daemonize: false,
+    debug: false,
+    logfile: LOGFILE,
+    minimum_age: 0,
+    monitor: false,
+    piddir: PIDDIR,
+    process_name: PROCESS_NAME,
+    redis: {
+      host: 'localhost',
+      port: 6379,
+      queue_name: 'dev:pmpy_queue'
+    },
+    # TODO: rest of these are examples for solr/fedora/swift... feel free to fill them in correctly
+    solr: {
+      url: 'http://localhost:8983/solr/hydra-development'
+    },
+    fedora: {
+      url: 'http://localhost:8984/rest',
```

Comment:
(In regards to the other comment ...)
Both HydraNorth and Avalon have 8983 as Fedora port in their configs. It's no biggie, and can be overridden, but it would be nicer to have a default that works out of the box.